app/helpers/navigation_helper.rb

```diff
@@ -33,9 +33,7 @@ def header_navigation
           { text: 'Artificial Intelligence', link: cms_page_path(page_slug: 'artificial-intelligence'), label: 'Artificial Intelligence' },
           { text: 'Pedagogy', link: cms_page_path(page_slug: '/pedagogy'), label: 'Pedagogy' },
           { text: 'Primary computing glossary', link: cms_page_path(page_slug: '/primary-computing-glossary'), label: 'Resources primary glossary' },
-          # The following are to be removed when Isaac page is added
-          { text: 'A level computer science', link: a_level_computer_science_path, label: 'A level computer science' },
-          { text: 'GCSE computer science', link: gcse_revision_path, label: 'GCSE revision' }
+          { text: 'Isaac Computer Science', link: about_isaac_computer_science_path, label: 'Isaac Computer Science' },
```

Comment:
Could the Isaac Computer Science be moved so that its below "Teaching resources"

https://github.com/NCCE/teachcomputing.org-issues/issues/2466 specifies that this menu should go in this order:
Teaching resources
Isaac Computer Science (when the page is merged)
Secondary question banks
Artificial intelligence
Pedagogy
Primary glossary (currently primary computing glossary)

Reply:
resolved too thank you